src/geom/SkyWcs.cc

```diff
@@ -334,7 +334,26 @@ std::shared_ptr<typehandling::Storable> SkyWcs::cloneStorable() const {
     return std::make_unique<SkyWcs>(*this);
 }
 
-std::string SkyWcs::toString() const { return "SkyWcs"; }
+std::string SkyWcs::toString() const {
+    std::ostringstream os;
+    if (isFits()) {
+        os << "FITS standard SkyWcs:";
+    } else {
+        os << "Non-standard SkyWcs (Frames: ";
+        // Print the frames in index order (frames are numbered from 1).
+        std::string delimiter = "";
+        for (size_t i = 1; i <= getFrameDict()->getAllDomains().size(); ++i) {
+            os << delimiter << getFrameDict()->getFrame(i)->getDomain();
+            delimiter = ", ";
+        }
+        os << "): ";
+    }
+    std::string delimiter = "\n";
+    os << delimiter << "Sky Origin: " << getSkyOrigin();
+    os << delimiter << "Pixel Origin: " << getPixelOrigin();
+    os << delimiter << "Pixel Scale: " << getPixelScale().asArcseconds() << " arcsec/pixel";
```

Comment:
Did you mean to have an `<< endl` after each statement?

Reply:
No, I intentionally put the delimiter first (in part so that whatever is produced in the `if isFits()` block is independent of the last three lines).

Reply:
Ok, I totally missed that. In the future, please use `endl` instead of `"\n"`. I realize the Stack is single-platform anyway, but it's the principle of the thing.

Reply:
`endl` and `\n` mean the same thing, except the former flushes the stream. Where you thinking of `\n` vs. `\r`?

Reply:
Aha, apparently getting the right line ending is the *stream*'s responsibility: https://stackoverflow.com/a/8033980. I didn't know that, never mind.